src/OpenAL/Silk.NET.OpenAL/AL/AL.cs

```diff
@@ -350,11 +349,33 @@ public static AL GetApi(bool soft = false)
             return ret;
         }
 
+        /// <summary>
+        ///     Attempts to load a native OpenAL extension of type <typeparamref name="T" />.
+        /// </summary>
+        /// <param name="ext">
+        ///     The loaded extension.
+        /// </param>
+        /// <typeparam name="T">
+        ///     Type of <see cref="NativeExtension{T}" /> to load.
+        /// </typeparam>
+        /// <returns>
+        ///     <c>True</c> if the extension was loaded, otherwise <c>False</c>.
+        /// </returns>
+        public bool TryGetExtension<T>(out T ext)
+            where T : NativeExtension<AL>
+        {
+            ext = IsExtensionPresent(ExtensionAttribute.GetExtensionAttribute(typeof(T)).Name)
+                ? (T) Activator.CreateInstance(typeof(T), Context)
+                : null;
+            return ext != null;
```

Comment:
Prefer identity comparisons
```suggestion
            return ext is not null;
```